corehq/util/test_utils.py

```diff
@@ -117,7 +117,7 @@ def get_json(cls, name, override_path=None):
 
     @classmethod
     def get_xml(cls, name, override_path=None):
-        return cls.get_file(name, '.xml', override_path)
+        return cls.get_file(name, '.xml', override_path).encode('utf-8')
```

Comment:
Is xml required to be a bytestring by something?

Reply:
Yes - our xml parsing library doesn't support unicode